### DISCUSSION TITLE
Welcome to Azure AI Developer Discussions!

### DISCUSSION THREAD

**Discussion by achandmsft** [Azure-Samples/azure-ai#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the Azure AI Developer community. An Azure AI developer is someone who wants to use Cognitive, Applied AI, OpenAI and other Azure AI APIs and SDKs to integrate AI into their apps. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StephenCleary** [comment]
Hi Arun!

I am Stephen Cleary, a .NET developer just getting started with AI. Excited to be here!

I haven't done anything yet, but I'm playing around with creating some .NET example code, and planning to write some blog posts on the experience.

**Comment by Strkli** [comment]
Hello all, I too am just getting started with AI and interested in how to/can implement AI into an application I'm building. Offhand I can identify the use of translation AI since the app will be global. Hopefully other use cases I'll be able to identify as I go along. This is such an exciting subject and time to get involved.

**Comment by Erenblack1107** [comment]
Hey!

I’ve seen Azure AI lately and exploring how it can power some of the [top AI apps](https://www.mobileappdaily.com/products/artificial-intelligence-apps?utm_source=github&utm_medium=referral&utm_campaign=om) out there. It’s been really cool to see how Cognitive Services and OpenAI APIs can be integrated to create smarter solutions. 

Excited to learn more from this community and see how you all are applying these tools in your projects!

Cheers
